api/services/pdf_auditor.py

```
@@ -0,0 +1,861 @@
+            self._audit_pdf_sync,
+            pdf_bytes,
+            filename
+        )
+
+    def _audit_pdf_sync(self, pdf_bytes: bytes, filename: str) -> PDFAuditReport:
+        """Auditoria síncrona (executada em thread separada)"""
+
+        try:
+            # Validações iniciais
+            if len(pdf_bytes) > self.MAX_FILE_SIZE:
+                raise ValueError(f"Arquivo excede {self.MAX_FILE_SIZE / 1024 / 1024}MB")
```

Ruff (0.14.4)

Abstract `raise` to an inner function

(TRY301)